QEfficient/exporter/export_hf_to_cloud_ai_100.py

```diff
@@ -465,7 +466,7 @@ def qualcomm_efficient_converter(
 
     """
     # Get model_kv first
-    model_kv = model_kv if model_kv else QEFFCommonLoader.from_pretrained(pretrained_model_name_or_path=model_name, hf_token=hf_token, cache_dir=cache_dir)
+    model_kv = model_kv if model_kv else QEFFCommonLoader.from_pretrained(pretrained_model_name=model_name, pretrained_model_path=model_path, hf_token=hf_token, cache_dir=cache_dir)
```

Comment:
Can't you directly use it like
if model_path not None:
```
model_kv = model_kv if model_kv else QEFFCommonLoader.from_pretrained(pretrained_model_name_or_path=(model_path if model_path else model_name), hf_token=hf_token, cache_dir=cache_dir)
```
I think since we are directly using HF APIs underneath this should work?